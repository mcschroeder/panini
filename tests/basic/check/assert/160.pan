assert : { b:𝔹 | b = true } → 𝟙

f160 : {x:bool|false} -> {v:unit|true}
f160 = \x:bool. assert x

f161 : {x:bool|false} -> {v:unit|false}
f161 = \x:bool. assert x
