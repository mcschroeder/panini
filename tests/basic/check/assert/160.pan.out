f160 : {x:𝔹 | false} → {v:𝟙 | true}
f161 : {x:𝔹 | false} → {v:𝟙 | false}